devbox.go

```diff
@@ -49,13 +49,26 @@ func Open(dir string) (*Devbox, error) {
 		return nil, errors.WithStack(err)
 	}
 
+	// if dir is current directory, then get the full path
```

Comment:
Instead of only resolving `.`, why not call `filpath.Abs` on any directory, to ensure everything is an absolute path?